src/ipld/util.rs

```diff
@@ -437,3 +438,259 @@ impl<DB: Blockstore, T: Iterator<Item = Tipset> + Unpin> Stream for ChainStream<
         }
     }
 }
+
+enum UnorderedTask {
```

Comment:
Any reason to use an `enum` here?

Reply:
It's a leftover, will remove it for good. Thanks for spotting this.